prime-router/src/testIntegration/kotlin/serializers/Hl7SerializerIntegrationTests.kt

```diff
@@ -452,6 +453,112 @@ NTE|1|L|This is a final comment|RE"""
         assertThat(result.report).isNull()
     }
 
+    @Test
+    fun `test cliaForOutOfStateTesting`() {
+        // arrange
+        val mcf = CanonicalModelClassFactory("2.5.1")
+        context.modelClassFactory = mcf
+        val parser = context.pipeParser
+        // act
+        val reg = "[\r\n]".toRegex()
+
+        val hl7Config = createConfig(cliaForOutOfStateTesting = "10DfakeCL")
+        val receiver = Receiver("test", "ca-dph", "covid-19", translation = hl7Config)
+        val schema = "direct/direct-covid-19"
+        val csvHeader = StringBuilder()
```

Comment:
Kotlin has a """ string construct that is easier to work with than StringBuilder.

Reply:
yeah that is easier :) Just made the change